app/src/main/kotlin/com/softteco/template/navigation/AppNavHost.kt

```diff
@@ -67,11 +72,23 @@ fun NavGraphBuilder.bottomBarGraph(navController: NavController) {
         composable(Screen.SignUp.route) {
             SignUpScreen(onBackClicked = { navController.navigateUp() })
         }
-        composable(route = Screen.ResetPassword.route) {
-            val token = "token" // TODO: deep link
+        composable(
+            route = Screen.ResetPassword.route,
+            deepLinks = listOf(
+                navDeepLink {
+                    uriPattern = Constants.RESET_PASSWORD_URL
+                    action = Intent.ACTION_VIEW
```

Comment:
A declaration in the intent filter is enough. If we declare an ```action``` here and the link doesn't contain the token argument, a crash will occur. Without this declaration, the main screen will open, which is fine